spec/helpers/jwt_auth_helpers_spec.rb

```diff
@@ -0,0 +1,112 @@
+require 'rails_helper'
+
+RSpec.describe JWTAuthHelpers, type: :helper do
+  describe '#user_info_from_header' do
+    it 'should return nil when the request does not have an Authorization header' do
+      allow(helper).to receive(:headers).and_return({})
+      expect(helper.send(:user_info_from_header)).to eq(nil)
```

Comment:
Hmm. Well in this particular case, the private method does rely on external entities (`AuthToken` and the `headers` method from `Grape::API`) and does use an instance variable. Would it best to just get rid of it completely and move the code into `current_user`? I think I broke it out separately because I was initially thinking that `authenticate_user!` would need to call it as well, but this turned out to be unnecessary.